_book/06-installing-or-building-from-sources/03-python.md

```diff
@@ -116,8 +116,6 @@ If you want to enable or disable other specific options, you can do:
 cmake ../cmake -B python -DBUILD_AS_PYTHON=ON -DNO_PAE_SUPPORT=ON
 ```
 
-*Installation with CMake has not be tested yet*
-
```

Comment:
You've tested it?

Reply:
Yes, tested on macOS and Ubuntu. That should be sufficient, I guess.

Reply:
Great. Where is the python module installed? You ran `sudo make install`?

Reply:
No, I built it in place and added the path to PYTHONOPATH. I can try to install it.

Reply:
OK, then some more information about that would be useful

Reply:
I've put it back in for now.